prime-router/start_func.sh

```diff
@@ -15,23 +15,12 @@ base_name=azure-functions/prime-data-hub-router
 # find the function folder
 if [ -d /prime-data-hub-router/$base_name ]; then
   function_folder=/prime-data-hub-router/$base_name
-elif [ -d target/$base_name ]; then
+elif [ -d build/$base_name ]; then
   function_folder=target/$base_name
 elif [ -d $base_name ]; then
```

Comment:
I think this line `function_folder=target/$base_name` should be `function_folder=build/$base_name`

Reply:
Yes, I must have changed it at some point.  Thx.